Source/BugsnagKSCrashSysInfoParser.m

```diff
@@ -85,17 +85,12 @@
     return appState;
 }
 
-NSDictionary *BSGParseAppState(NSDictionary *report) {
-    NSDictionary *system = report[BSGKeySystem];
+NSDictionary *BSGParseAppState(NSDictionary *report, NSString *preferredVersion) {
     NSMutableDictionary *app = [NSMutableDictionary dictionary];
 
-    NSString *version = [report valueForKeyPath:@"user.config.appVersion"];
-
-    if (!version) {
-        version = system[@"CFBundleShortVersionString"];
-    }
+    NSString *version = preferredVersion ?: report[@"CFBundleShortVersionString"];
 
-    BSGDictSetSafeObject(app, system[@"CFBundleVersion"], @"bundleVersion");
+    BSGDictSetSafeObject(app, report[@"CFBundleVersion"], @"bundleVersion");
```

Comment:
should `bundleVersion` be a const like the other keys? `BSGKeyReleaseStage` etc.

Reply:
Maybe; I want to think about how we are doing key constants a bit in the meantime in case there's an easy way to both define and find the right ones.